controller/lib/hal/adc.cpp

```diff
@@ -54,24 +54,18 @@ limitations under the License.
 #include "hal.h"
 #include "hal_stm32.h"
 
-/******************************************************************
- * A/D inputs.
- *
- * The following pins are used as analog inputs on the rev-1 PCB
- *
- * Please refer to the PCB schematic as the ultimate source of which
- * pin is used for which function.  A less definitive, but perhaps
- * easier to read spreadsheet is availabe here:
- * https://docs.google.com/spreadsheets/d/1JOSQKxkQxXJ6MCMDI9PwUQ6kiuGdujR4D6EJN9u2LWg/edit#gid=0
- *
- * PA0 (ADC1_IN5)  - vin (not currently used)
- * PA1 (ADC1_IN6)  - pressure
- * PA4 (ADC1_IN9)  - inhale flow
- * PB0 (ADC1_IN15) - exhale flow
- *
- * See chapter 16 of the reference manual
- *
- *****************************************************************/
+/*
+The following pins are used as analog inputs on the rev-1 PCB:
+- PA0 (ADC1_IN5)  - vin (not currently used)
+- PA1 (ADC1_IN6)  - pressure
+- PA4 (ADC1_IN9)  - inhale flow
+- PB0 (ADC1_IN15) - exhale flow
+
+Please refer to the PCB schematic as the ultimate source of which
+pin is used for which function.
```

Comment:
Perhaps put the last paragraph first, and then you can say here, "Please refer to [PCB] as...".  That way I'm not left wondering where to find the PCB schematic.